rebar.config

```diff
@@ -24,29 +24,27 @@
 
 {deps, [{lager, ".*", {git, "https://github.com/erlang-lager/lager.git",
                       {ref, "69b4ada"}}}, % tag: 3.6.7
-        {setup, "2.0.2"},
-        {parse_trans, "3.2.0"},
         {mnesia_rocksdb, ".*", {git, "https://github.com/aeternity/mnesia_rocksdb.git", {ref,"ad8e7b6"}}},
         {enacl, ".*", {git, "https://github.com/jlouis/enacl.git", {ref,"885662c"}}},
         {enoise, "1.0.0"},
         {cowboy, "2.2.2", {git, "https://github.com/ninenines/cowboy.git", {tag, "2.2.2"}}},
-        {jsx, {git, "https://github.com/talentdeficit/jsx.git", {tag, "2.8.0"}}},
-        {jesse, {git, "https://github.com/aeternity/jesse.git", {ref, "4640ad1"}}}, %% Tag `1.5.0-rc2` with workaround for unexported `http_uri:uri()` type.
+        {jesse, {git, "https://github.com/tolbrino/jesse.git",
+                {ref, "b0a3bae"}}}, % tag 1.5.2 + http_uri patch
         {gproc, "0.6.1"},
         {jobs, "0.9.0"},
-        {exometer_core, {git, "https://github.com/Feuerlabs/exometer_core.git", {ref, "f088de5"}}},
+        {exometer_core, "1.5.7"},
         {poolboy, "1.5.1"},
         {yamerl, "0.7.0"},
         {base58, {git, "https://github.com/aeternity/erl-base58.git", {ref,"60a3356"}}},
-        {eper, ".*", {git, "git://github.com/aeternity/eper.git", {ref, "03d4be3"}}},
+        {eper, "0.99.1"},
         {erlexec, ".*", {git, "https://github.com/saleyn/erlexec.git", {ref, "97a5188"}}},
         {sha3, {git, "https://github.com/szktty/erlang-sha3", {ref, "dbdfd12"}}},
         {sext, {git, "https://github.com/uwiger/sext.git", {ref, "615eebc"}}},
         {idna, {git, "https://github.com/aeternity/erlang-idna", {tag, "6.0.0"}}}
        ]}.
 
 %% pc needed for erlexec and version 1.6.0 works, but 1.9.1 does not.
```

Comment:
As the plugin is not used anymore (?) I guess you could remove this commend as well

Reply:
Done.